public/v4/apps/transmission.yml

```diff
@@ -0,0 +1,63 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: ghcr.io/linuxserver/transmission:$$cap_transmission_version
+        environment:
+            PUID: '1000'
+            PGID: '1000'
+            TZ: $$cap_tz
+            TRANSMISSION_WEB_HOME: /$$cap_ui/
+            USER: $$cap_user
+            PASS: $$cap_pass
+            WHITELIST: '' #optional
+            HOST_WHITELIST: '' #optional
+            WEBUI_PORT: '9091'
+        ports:
+            - 9091:9091
```

Comment:
You probably don't need to port map 9091 as it's already being using proxied through nginx.

Reply:
sure